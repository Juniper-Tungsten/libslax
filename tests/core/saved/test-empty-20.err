done: 17
done: 17
done: 17
