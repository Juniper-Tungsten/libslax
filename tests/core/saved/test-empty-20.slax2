/* Machine Crafted with Care (tm) by slaxWriter */
version 1.1;

var $content = "\nline one\nline two\nline three\nline four\nline five\nse-ix\nse-ven\neh-it\nny-en\nte-en\nel-e-ven\ntwiel-ve\nt'ir-teeen\nfour-teen\nfift-teen\nse-ix-teen\n";
var $content2 = <this> $content;

match / {
    <top> {
        var $lines = slax:break-lines($content);
        
        message "done: " _ count($lines);
        
        for $line ($lines) {
            copy-of $line;
        }
        var $lines2 = slax:break-lines($content2);
        message "done: " _ count($lines);
        
        for $line2 ($lines2) {
            copy-of $line2;
        }
    }
}
