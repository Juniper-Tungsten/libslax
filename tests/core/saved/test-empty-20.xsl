<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:slax="http://xml.libslax.org/slax" version="1.0" extension-element-prefixes="slax">
  <xsl:variable name="content" select="&quot;&#10;line one&#10;line two&#10;line three&#10;line four&#10;line five&#10;se-ix&#10;se-ven&#10;eh-it&#10;ny-en&#10;te-en&#10;el-e-ven&#10;twiel-ve&#10;t'ir-teeen&#10;four-teen&#10;fift-teen&#10;se-ix-teen&#10;&quot;"/>
  <xsl:variable name="content2">
    <this>
      <xsl:value-of select="$content"/>
    </this>
  </xsl:variable>
  <xsl:template match="/">
    <top>
      <xsl:variable xmlns:slax="http://xml.libslax.org/slax" name="lines" select="slax:break-lines($content)"/>
      <xsl:message>
        <xsl:value-of select="concat(&quot;done: &quot;, count($lines))"/>
      </xsl:message>
      <xsl:variable name="slax-dot-1" select="."/>
      <xsl:for-each select="$lines">
        <xsl:variable name="line" select="."/>
        <xsl:for-each select="$slax-dot-1">
          <xsl:copy-of select="$line"/>
        </xsl:for-each>
      </xsl:for-each>
      <xsl:variable xmlns:slax="http://xml.libslax.org/slax" name="lines2" select="slax:break-lines($content2)"/>
      <xsl:message>
        <xsl:value-of select="concat(&quot;done: &quot;, count($lines))"/>
      </xsl:message>
      <xsl:variable name="slax-dot-2" select="."/>
      <xsl:for-each select="$lines2">
        <xsl:variable name="line2" select="."/>
        <xsl:for-each select="$slax-dot-2">
          <xsl:copy-of select="$line2"/>
        </xsl:for-each>
      </xsl:for-each>
    </top>
  </xsl:template>
</xsl:stylesheet>
