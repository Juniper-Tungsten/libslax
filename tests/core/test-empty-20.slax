version 1.1;

var $content = "
line one
line two
line three
line four
line five
se-ix
se-ven
eh-it
ny-en
te-en
el-e-ven
twiel-ve
t'ir-teeen
four-teen
fift-teen
se-ix-teen
";

var $content2 = <this> $content;

match / {
    <top> {
	var $lines = slax:break-lines($content);
	message "done: " _ count($lines);
	for $line ($lines) {
	    copy-of $line;
	}
	var $lines2 = slax:break-lines($content2);
	message "done: " _ count($lines);
	for $line2 ($lines2) {
	    copy-of $line2;
	}
    }
}
