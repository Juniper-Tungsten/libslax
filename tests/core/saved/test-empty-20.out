<?xml version="1.0"?>
<top>
  <text></text>
  <text>line one</text>
  <text>line two</text>
  <text>line three</text>
  <text>line four</text>
  <text>line five</text>
  <text>se-ix</text>
  <text>se-ven</text>
  <text>eh-it</text>
  <text>ny-en</text>
  <text>te-en</text>
  <text>el-e-ven</text>
  <text>twiel-ve</text>
  <text>t'ir-teeen</text>
  <text>four-teen</text>
  <text>fift-teen</text>
  <text>se-ix-teen</text>
  <this></this>
  <this>line one</this>
  <this>line two</this>
  <this>line three</this>
  <this>line four</this>
  <this>line five</this>
  <this>se-ix</this>
  <this>se-ven</this>
  <this>eh-it</this>
  <this>ny-en</this>
  <this>te-en</this>
  <this>el-e-ven</this>
  <this>twiel-ve</this>
  <this>t'ir-teeen</this>
  <this>four-teen</this>
  <this>fift-teen</this>
  <this>se-ix-teen</this>
</top>
